scripts/backfill_participant_mappings/main.go

```
@@ -0,0 +1,405 @@
+		if uid == "" || uid == entry.Key() {
+			slog.WarnContext(ctx, "unexpected mapping key format, skipping", "key", entry.Key())
+			skipped++
+			continue
+		}
+		legacy = append(legacy, legacyEntry{
+			mappingKey:   entry.Key(),
+			currentValue: currentValue,
+			uid:          uid,
+		})
+	}
+	watcher.Stop()
```

golangci-lint (2.12.2)

[error] Error return value of `watcher.Stop` is not checked

(errcheck)